client.go

```diff
@@ -248,7 +248,7 @@ func (c *dedicatedSingleClient) release() {
 }
 
 func (c *singleClient) isRetryable(err error, ctx context.Context) bool {
-	return err != nil && atomic.LoadUint32(&c.stop) == 0 && ctx.Err() == nil
+	return err != nil && atomic.LoadUint32(&c.stop) == 0 && ctx.Err() == nil && err != ErrDoCacheAborted
```

Comment:
Hi @ajayd-san, thanks! Could you change the order of the new check? Move it to the in front of the atomic.LoadUint32.

Reply:
Done!